### PR TITLE
feat(GRO-736): Pages

### PR DESCRIPTION
**Pull request by dzucconi**
The type of this PR is: **Feature**

This PR solves [GRO-736]

Implements new page app. Removes the old one.

#### New / Old

[GRO-736]: https://artsyproduct.atlassian.net/browse/GRO-736?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/Apps/Page/pageRoutes.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "auction-info",
+  "embed-terms",
+  "past-terms-8-5-13",
+  "past-privacy-10-29-12",
+  "rrf-emerging-curator-competition-official-rules",
+  "past-privacy-8-5-13",
+  "past-terms-9-26-13",
+  "buy-now-feature-faq",
+  "identity-verification-faq",
+]
```
Comment (dzucconi) [inline src/v2/Apps/Page/pageRoutes.tsx:30]: We support these slugs as top-level style routes. e.g. `artsy.net/terms`

File: src/desktop/assets/misc.styl [hunk 2]
```diff
@@ -11,8 +11,6 @@
 // jobs
 @require '../components/chevron_list';
 @require '../apps/jobs/stylesheets';
-// page
-@require '../apps/page';
```
Comment (jonallured) [inline src/desktop/assets/misc.styl:15]: You love to see it!

File: src/lib/middleware/hardcodedRedirects.ts [hunk 3]
```diff
@@ -82,6 +82,7 @@ const redirects = {
   "/user/purchases": "/settings/purchases",
   "/user/payments": "/settings/payments",
   "/user/alerts": "/settings/alerts",
+  "/page/collector-faqs-selling-on-artsy": "/consign",
```
Comment (jonallured) [inline src/lib/middleware/hardcodedRedirects.ts:85]: How did you know to do this?

File: src/v2/Utils/toStyle.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export const toStyle = (style: Record<string, string | number | undefined>) => {
+  return Object.entries(style)
+    .map(([key, value]) => {
+      const property = key.replace(/[A-Z]/g, m => `-${m.toLowerCase()}`)
+      return `${property}: ${value};`
```
Comment (jonallured) [inline src/v2/Utils/toStyle.ts:5]: But wait how can I mark my styles as IMPORTANT Damon??

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

File: src/lib/middleware/hardcodedRedirects.ts (hunk 3 above)
Comment (dzucconi) [reply src/lib/middleware/hardcodedRedirects.ts:85]: https://github.com/artsy/force/blob/3ebd208040dc8d89da0acd6add8aacc6e3320ae8/src/desktop/apps/page/routes.coffee#L11-L12

**Pull request merged**